Patch 1

phorn30 opened the pull request:
Beefed up the Azure Migrate section and added a new "What Should I Use?" section based on our call on 10/24.

phorn30 committed:
Update migrate-overview.md

Added clarity to recommendations.

phorn30 committed:
Update migrate-overview.md

brsteph commented on articles/site-recovery/migrate-overview.md:
```diff
@@ -34,7 +36,20 @@ Site Recovery should be used:
+In sum:
+
+- If you are beginning a new migration with neither ASR nor Migrate in place, use Azure Migrate.
+- If you have ASR in place actively protecting machines, continue to use it.
```
Can we adjust this to "continue to use it for replication, but consider using Azure Migrate for business cases and dependency analysis" or something like that?

brsteph replied:
@phorn30 ^ in case the above didn't notify you.

phorn30 committed:
Update migrate-overview.md

Added clarity to help customer choose which one to use.  Added a suggested enhancement to one of the summary comments.

ankitaduttaMSFT commented on articles/site-recovery/migrate-overview.md:
```diff
@@ -34,7 +36,20 @@ Site Recovery should be used:
+
+We recommend using Azure Migrate to migrate on-premises servers to Azure.  However, if you've already started your migration journey with Site Recovery, consider the following:
+
+- With ASR already replicating your servers, there is no need to deploy a Migrate appliance, remove the BCDR protection, and replicate with a new appliance.
```
@phorn30, pls update "ASR" to "Azure Site Recovery"
